src/ol_infrastructure/applications/ocw_site/snippets/robots.production.txt

```diff
@@ -0,0 +1,2 @@
+User-Agent: *
+Allow: /
```

Comment:
```suggestion
Allow: /
Sitemap: https://ocw.mit.edu/sitemap.xml
```

We had a disallow on `/ans78070/` in the earlier version, but I'd like to experiment with remove it.